kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/QueryDataServiceIntegrationTest.java

```diff
@@ -917,12 +917,12 @@ public void testGetTasksWithPotentialOwnerDoubleGroup() throws Exception {
     public void testGetProcessInstanceCustomMapper() throws Exception {
         Map<String, Object> parameters = new HashMap<String, Object>();
         parameters.put("stringData", "waiting for signal");
-        parameters.put("personData", createPersonInstance(CONTAINER_ID));
+        parameters.put("personData", USER_JOHN);
```

Comment:
Wouldn't it be better to just leave it as is and just compare it at the end that it is equals to toString representation of the object.